src/index.ts

```diff
@@ -185,6 +185,19 @@ function isObject(value: unknown) {
     return value != null && (type === 'object' || type === 'function')
 }
 
+// See https://github.com/quanticedu/back_royal/pull/10744#issuecomment-1287033565
+// FIXME: if we decide to merge this change, we should add non-standard custom providers
+// (apple_quantic, apple_smartly, wechat_web, wechat_native and wechat_official_account) to DeviseTokenAuthClient instance initialization
+// in front-royal `devise_token_auth_client_module.js` and `devise-token-auth-client.ts` in Gatsby to support apple and wechat login
+export const AuthProviderPathsDefaultConfig = {
+    google_oauth2: '/auth/google_oauth2',
+    facebook: '/auth/facebook',
+    apple: '/auth/apple',
+    twitter: '/auth/twitter',
+    onelogin: '/auth/onelogin',
+    wechat: '/auth/wechat',
```

Comment:
Maybe we should mimic the same logic from the original [package code](https://github.com/lynndylanhurley/ng-token-auth/blob/c2b6e788198fd09f9922952c188aa70f790434b5/src/ng-token-auth.coffee#L51-L55) in order to be a proper replacement for it.